docs/content/installation/install_driver_github.md

```diff
@@ -2,7 +2,7 @@
 
 The operator for IBM® block storage CSI driver can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
```

Comment:
```suggestion
The IBM® block storage CSI driver operator can be installed directly with GitHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
```

Reply:
I don't see a need to update this. i think that the way it is now is clearer